core/commands/p2p.go

```diff
@@ -49,6 +51,16 @@ type P2PStreamsOutput struct {
 	Streams []P2PStreamInfoOutput
 }
 
+const (
+	allowCustomProtocolOptionName = "allow-custom-protocol"
+	allOptionName                 = "all"
+	protocolOptionName            = "protocol"
+	listenAddressOptionName       = "listen-address"
+	targetAddressOptionName       = "target-address"
+)
+
+var connectionTimeout = 10 * time.Second
```

Comment:
This doesn't appear to be used anywhere but `parseIpfsAddr`, so maybe call it `resolveTimeout`?